### DISCUSSION TITLE
El-Switch CSS样式被破坏

### DISCUSSION THREAD

**Discussion by AaronConlon** [element-plus/element-plus#10609] [Q&A]
你好，我遇到了一个CSS的问题。
我在项目里使用了`El-Switch`组件，一开始一切正常，但是或许是有人增加了破坏性的`CSS`规则，让这个组件在切换状态的时候被破坏了原样式。
如下是两个状态切换后的结果：
<img width="66" alt="image" src="https://user-images.githubusercontent.com/23721611/201824405-e411f817-a334-43c4-9f9e-b48610f2691b.png">
<img width="61" alt="image" src="https://user-images.githubusercontent.com/23721611/201824424-29ea0fde-6fe5-4b6f-a580-61d25fa7e920.png">

大家有什么思路吗？😂 我实在找不到思路，因此到这里寻求帮助。感谢

**Comment by webvs2** [comment]
I will check for you, please provide the minimum instance of the problem code

**Comment by ryuhangyeong** [comment]
We need examples to help!

**Comment by AaronConlon** [comment]
wait a moment...
